data/src/main/kotlin/com/vultisig/wallet/data/common/FileHelper.kt

```
@@ -69,6 +72,27 @@ fun Context.saveContentToUri(uri: Uri, content: String): Boolean {
+                contentList.forEach { content ->
+                    val zipEntry = ZipEntry(content.name)
+                    zipOutputStream.putNextEntry(zipEntry)
+                    content.content.byteInputStream().use { input ->
+                        input.copyTo(zipOutputStream, DEFAULT_BUFFER_SIZE)
+                    }
+                    zipOutputStream.closeEntry()
+                }
+            }
+        }
+        return true
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)